### DISCUSSION TITLE
GSMS not detecting games that are scanned via Nvidia App

### DISCUSSION THREAD

**Discussion by SnappierSoap318** [LizardByte/community#537] [GSMS] [answered]
### Select Topic Area

Bug

### Body

(not sure if it's a bug or not?) 

gsms doesn't find games that was scanned by the new Nvidia App. 

Any work around?

Nvidia App scanned list: 
![image](https://github.com/user-attachments/assets/c023cc24-e8cd-4d6b-96ba-9744d4f95608)

Nvidia App Version:
![image](https://github.com/user-attachments/assets/ef5fe0b4-c338-482b-88e3-7e4c9512c2b2)

Winver: 
![image](https://github.com/user-attachments/assets/6466bbf2-1a29-4fa1-aafe-102d64332dc8)

GSMS log: 
```txt
PS C:\Users\adwai\Downloads> ./gsms.exe --dry_run
Found apps.json file.
{
    "env": {},
    "apps": [
        {
            "name": "Desktop",
            "image-path": "desktop.png"
        },
        {
            "name": "Steam Big Picture",
            "cmd": "steam://open/bigpicture",
            "auto-detach": "true",
            "wait-all": "true",
            "image-path": "steam.png"
        }
    ]
}
----
Traceback (most recent call last):
  File "gsms\gsms.py", line 525, in <module>
  File "gsms\gsms.py", line 417, in main
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\adwai\\AppData\\Local\\NVIDIA Corporation\\Shield Apps'
[12636] Failed to execute script 'gsms' due to unhandled exception!
```

**Comment by ReenigneArcher** [comment, marked as answer]
GSMS is Gamestream migration service. It was intended to help people migrate from GFE Gamestream to Sunshine.

Edit: It was never intended to work with the new Nvidia App, but PRs would be accepted to extend the functionality. There is also a game finder (for Steam and Epic games) on the Sunshine roadmap.
